pkg/azureclients/privatelinkserviceclient/azure_privatelinkserviceclient_test.go

```diff
@@ -319,6 +350,13 @@ func getTestPrivateLinkService(name string) network.PrivateLinkService {
 	}
 }
 
+func getTestPrivateEndpointConnection(PLSName string, PEConnName string) network.PrivateEndpointConnection {
+	return network.PrivateEndpointConnection{
+		ID:   to.StringPtr(fmt.Sprintf("/subscriptions/subscriptionID/resourceGroups/rg/providers/"+PLSResourceType+"/%s/%s/%s", PLSName, PEConnResourceType, PEConnName)),
```

Comment:
Why not like this? Current one seems a bit strange :confused:
```suggestion
		ID:   to.StringPtr(fmt.Sprintf("/subscriptions/subscriptionID/resourceGroups/rg/providers/%s/%s/%s/%s", PLSResourceType, PLSName, PEConnResourceType, PEConnName)),
```

Reply:
I was following the above `getTestPrivateLinkService` function. But I don't see any problem changing it.

Reply:
I think all of they can be changed.